Fix polling interval docs (#4988)
quickwit-oss/quickwit: docs/configuration/metastore-config.md

You can also configure it to poll the File-Backed Metastore periodically to keep a fresh view of it. This is useful for a Searcher instance that needs to be aware of new splits published by an Indexer running in parallel.

> To configure the polling interval (in seconds), add a URI fragment to the storage URI like this: `s3://quickwit/my-indexes#polling_interval=30s`

Reviewer: ```suggestion
To configure the polling interval (in seconds), add a URI fragment to the storage URI as follows: `s3://quickwit/my-indexes#polling_interval=30s`
```

:::note

> The polling interval can be configured in seconds only, other units like minutes or hours are not supported yet.

Reviewer: ```suggestion
The polling interval can be configured in seconds only; other units, such as minutes or hours, are not supported.
```
